tests/acceptance/features/apiGraph/getUser.feature

```diff
@@ -46,28 +98,160 @@ Feature: get users
     Given the administrator has given "Alice" the role "Admin" using the settings api
     When user "Alice" gets all users using the Graph API
     Then the HTTP status code should be "200"
-    And the API response should contain following users with the information:
-      | displayName  | id        | mail              | onPremisesSamAccountName | accountEnabled |
-      | Alice Hansen | %uuid_v4% | alice@example.org | Alice                    | true           |
-      | Brian Murphy | %uuid_v4% | brian@example.org | Brian                    | true           |
+    And the JSON data of the response should contain the user "Alice Hansen" in the item 'value', the user-details should match
```

Comment:
is it possible to check both users in the one step? 
`the JSON data of the response should contain the users "Alice Hansen, Brian Murphy" in the item 'value', the user-details should match`

Reply:
The schema has a `array contains` property but it seems to be a bug in the library. sadly we must filter what to check when asserting.